<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd">
<suite name="Failed suite [restAssuredTestSuite]">
  <test thread-count="5" name="restAssuredTests(failed)">
    <classes>
      <class name="TestCases.TC004_Update_Employee">
        <methods>
          <include name="checkContentType"/>
          <include name="setUp"/>
          <include name="checkResponseBody"/>
          <include name="checkStatusCode"/>
          <include name="updateEmployee"/>
        </methods>
      </class> <!-- TestCases.TC004_Update_Employee -->
      <class name="TestCases.TC002_create_New_Employee">
        <methods>
          <include name="checkContentType"/>
          <include name="checkStatusCode"/>
          <include name="createEmployee"/>
          <include name="checkResponseBody"/>
          <include name="setUp"/>
        </methods>
      </class> <!-- TestCases.TC002_create_New_Employee -->
      <class name="TestCases.TC001_Get_All_Employees">
        <methods>
          <include name="checkContentType"/>
          <include name="getAllEmployees"/>
          <include name="setUp"/>
          <include name="tearDown"/>
          <include name="checkStatusCode"/>
        </methods>
      </class> <!-- TestCases.TC001_Get_All_Employees -->
    </classes>
  </test> <!-- restAssuredTests(failed) -->
</suite> <!-- Failed suite [restAssuredTestSuite] -->
